docs/install.md

```diff
@@ -230,6 +230,24 @@ SHARP_IGNORE_GLOBAL_LIBVIPS=1 npm install --arch=x64 --platform=linux --libc=gli
 To get the best performance select the largest memory available.
 A 1536 MB function provides ~12x more CPU time than a 128 MB function.
 
+### serverless-esbuild
+
+To deploy AWS Lambda using serverless framework, esbuild and serverless-esbuild on machines other than Linux x64 (glibc), first ensure sharp is excluded from bundling via the
+[exclude option](https://www.serverless.com/plugins/serverless-esbuild#options)
+configuration.
+Make changes inside `serverless.yml` under `esbuild` with the following.
+```
+esbuild:
+    # Keep existing changes
```

Comment:
Is the YAML indentation correct here? Is a `custom` parent required, as shown in https://www.serverless.com/plugins/serverless-esbuild#options ?

Reply:
The esbuild configurations go under custom, thought it would be obvious for someone who is using esbuild to know about it. Although, changed it to proper YAML indentation just in case of any misunderstanding.